### ISSUE TITLE
SSH Access

### ISSUE THREAD

**Issue by obsy** [ericpaulbishop/gargoyle#477]
```
# uci show dropbear
dropbear.@dropbear[0]=dropbear
dropbear.@dropbear[0].PasswordAuth='on'
dropbear.@dropbear[0].RootPasswordAuth='on'
dropbear.@dropbear[0].Port='22'
```

dropbear has unnamed section. So 
```
+	sshPwdEnabled = uciOriginal.get("dropbear", "global", "PasswordAuth") == "on" ? true : false;
```
is wrong, because 'global' section does not exist

**Comment by ericpaulbishop** [comment]
If it can't find the section, it defaults to false, which is true anyway if it hasn't been enabled by the interface. But, when saving access.js the section is renamed to global. 

So, if it's off anyway, that is correctly reflected. If it has been set, then the section will be properly renamed.

**Comment by obsy** [comment]
On default installation PasswordAuth is enabled by default and option in gui is NOT checked. Should be.

**Comment by ericpaulbishop** [comment]
Ah, good point. Fixing it now.

On Sat, Feb 27, 2016 at 4:50 PM, Cezary Jackiewicz <notifications@github.com

**Comment by ericpaulbishop** [comment]
Should now be fixed in 2f0f7cd234733e3579bfbf70d76d4a7bf2044a1e
